js/apps/admin-ui/CONTRIBUTING.md

```diff
@@ -6,7 +6,7 @@ This project is the next generation of the Keycloak Administration UI. It is wri
 
 ### Prerequisites
 
-Make sure that you have Node.js version 18 (or later) installed on your system. If you do not have Node.js installed we recommend using [Node Version Manager](https://github.com/nvm-sh/nvm) to install it.
+Make sure that you have Node.js version 20.19+ or 22.12+ (or later) installed on your system. If you do not have Node.js installed we recommend using [Node Version Manager](https://github.com/nvm-sh/nvm) to install it.
```

Comment:
Since this will always be a moving target, and the Keycloak team strives to always support at least the latest LTS, let's not use exact numbers here.
```suggestion
Make sure that you have the latest LTS version of Node.js (or newer) installed on your system. If you do not have Node.js installed we recommend using [Node Version Manager](https://github.com/nvm-sh/nvm) to install it.
```

Reply:
Having a specific version which a contributor should use might be helpful, especially if they would be in a release branch. Would point towards one of your GitHub workflows where they could find that version if we don't want to reference it by the version number here?